Jenkinsfile

```diff
@@ -40,13 +40,50 @@ def allArchitectures = [
 ]
 jobProperties.add(parameters([text(defaultValue: allArchitectures.join('\n'),
         description: 'The architectures (cheribuild suffixes) to build for (one per line)',
-        name: 'architectures')]))
+        name: 'architectures'),
+        text(defaultValue: ["riscv64-hybrid", "riscv64-purecap"].join('\n'),
+        description: 'The architectures (cheribuild suffixes) to build a purecap kernel for (one per line)',
+        name: 'purecapKernelArchitectures')]))
 // Set the default job properties (work around properties() not being additive but replacing)
 setDefaultJobProperties(jobProperties)
 
 jobs = [:]
 
-def buildImageAndRunTests(params, String suffix) {
+def selectedPurecapKernelArchitectures = params.purecapKernelArchitectures.split('\n')
```

Comment:
Groovy's scoping rules are stupid so this won't work.

You'll need to add `public static List<String> selectedPurecapKernelArchitectures = []` to `GlobalVars` as a possible workaround.

Reply:
:(